src/main/java/ortus/boxlang/runtime/jdbc/ExecutedQuery.java

```diff
@@ -183,21 +306,26 @@ public static ExecutedQuery fromPendingQuery( @NonNull PendingQuery pendingQuery
 		results.setMetadata( queryMeta );
 		ExecutedQuery executedQuery = new ExecutedQuery( results, generatedKey );
 
-		interceptorService.announce(
-		    BoxEvent.POST_QUERY_EXECUTE,
-		    Struct.of(
-		        "sql", queryMeta.getAsString( Key.sql ),
-		        "bindings", pendingQuery.getParameterValues(),
-		        "executionTime", executionTime,
-		        "data", results,
-		        "result", queryMeta,
-		        "pendingQuery", pendingQuery,
-		        "executedQuery", executedQuery
-		    )
-		);
+		interceptorService.announce( BoxEvent.POST_QUERY_EXECUTE,
+		    Struct.of( "sql", queryMeta.getAsString( Key.sql ), "bindings", pendingQuery.getParameterValues(), "executionTime", executionTime, "data", results,
+		        "result", queryMeta, "pendingQuery", pendingQuery, "executedQuery", executedQuery ) );
 		return executedQuery;
 	}
 
+	private static Object processGeneratedKeys( ResultSet rs, Array allGeneratedKeys, Object generatedKey ) throws SQLException {
+		// System.out.println( "retrieving generated keys posing as a result set" );
```

Comment:
Could you take this comment and put it in a docblock describing the method just so I know what the purpose of "processGeneratedKeys" is when I come across it next week/month?

Reply:
Oh, oops, I completely missed the java docs on that.  That was some logic I had duplicated twice above and I tried to move it into a lambda, but Java is stupid so I moved it down to a static method, but just missed the comment block at the top.